InfluxDB.Net/InfluxDB.Net.csproj

```diff
@@ -13,8 +13,8 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <TargetFrameworkProfile>Profile344</TargetFrameworkProfile>
```

Comment:
@ziyasal I couldn't make the code work with GzipStream and i readded Microsoft Http client libraries nuget packages which caused target profile change. How bad is this change, in your opinion? Also, do we need a full backward-compatibility with 0.8 and earlier?